guides/common/modules/proc_configuring-the-fallback-to-any-smartproxy-remote-execution-setting-in-project.adoc

```diff
@@ -26,6 +26,6 @@ To set the value to `true`, enter the following command:
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 $ hammer settings set \
---name=remote_execution_fallback_proxy \
---value=true
+--name remote_execution_fallback_proxy \
+--value true
```

Comment:
Technically this isn't an installer option so the PR description isn't entirely correct. I don't mind updating those too, but consider splitting it.

Reply:
I have updated the PR to match the change+commit message.